### PR TITLE
[Github workflow] update format

### PR DESCRIPTION
**Pull request by Jim8y**
only run dotnet format on changed file to save the dotnet format time

### CODE REVIEW

**Comment by Jim8y** [comment]
Why having this pr? I am working to fix the run only failed jobs, turns out having multiple sub jobs will increase the workflow executing time, making it necessary to optimize the time as many as possible. This pr saves 50% of the format time.

File: tests/Neo.Compiler.CSharp.UnitTests/UnitTest_MultipleB.cs [hunk 1]
```diff
@@ -12,5 +12,11 @@ public void Test()
             Assert.IsFalse(Contract.Test());
             Assert.AreEqual(984060, Engine.FeeConsumed.Value);
         }
+
+        [TestMethod]
+        public void Test2()
+        {
+            // Test if the workflow format is working
+        }
```
Comment (Jim8y) [inline tests/Neo.Compiler.CSharp.UnitTests/UnitTest_MultipleB.cs:20]:
```suggestion
```

**Review by shargon — COMMENTED** [review]
Json is not checked?

**Comment by Jim8y** [comment]
> Json is not checked?

do we have json in devpack?

**Comment by shargon** [comment]
> > Json is not checked?
> 
> do we have json in devpack?

No, but this will be copied to neo?

**Comment by Jim8y** [comment]
> > > Json is not checked?
> > 
> > do we have json in devpack?
> 
> No, but this will be copied to neo?

sure, i will update later, can you help to check the coverage failure in other prs? i was trying to fix that problem thus having this one.

**Review by shargon — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**